langfuse/src/openai.ts

```diff
@@ -0,0 +1,256 @@
+import OpenAI from "openai";
+import { Langfuse, LangfuseTraceClient } from "./langfuse";
+
+
+const client = new Langfuse();
+interface InputArgsData {
+    model: string
+    input: Record<string, any>
+    modelParams: Record<string, any>
+}
+
+const parseInputArgs = (args: Record<string, any>): InputArgsData => {
```

Comment:
could map tools / toolchoice etc to metadata

Reply:
functions, function_call, tools and tool_choice are being sent as part of the input right now. Should it remain the same and additionally send it in the metadata as well ? 
Or should it be sent only in metadata. ?

Reply:
The UI doesn't display tools and tool_choice is regular dislay, but in the json mode it is visible.

Reply:
In input is correct, need to fix the front-end rendering when tools are on top-level, currently only rendered well when they are within a single message.

Makes sense to not make them to metadata, agree!